src/pages/viewer-app/viewer-app.tsx

```diff
@@ -145,10 +145,14 @@ export const ViewerApp = () => {
   const [loadedTilesets, setLoadedTilesets] = useState<Tileset3D[]>([]);
 
   const initMainTileset = () => {
```

Comment:
I would suggest to change it like paste below:
```
const initMainTileset = () => {
    const tilesetParam = parseTilesetFromUrl();

    if (tilesetParam?.startsWith("http")) {
      return {
        id: tilesetParam,
        name: CUSTOM_EXAMPLE_VALUE,
        url: tilesetParam,
      };
    }

    if (tilesetParam in EXAMPLES) {
      return EXAMPLES[tilesetParam];
    }

    return EXAMPLES[INITIAL_EXAMPLE_NAME];
  };
```

And also in parseTileset() method:

`export const parseTilesetFromUrl = () => {
  const parsedUrl = new URL(window.location.href);
  return parsedUrl.searchParams.get("tileset") || '';
};
`